kubetest/gke.go

```diff
@@ -608,14 +610,17 @@ func (g *gkeDeployer) setupBastion() error {
 	filtersToTry = append(filtersToTry, exactFilter)
 	// As a fallback - use proxy instance name as a regex but check only cluster nodes
 	var igFilters []string
+	// filter out VMs not belonging to the GKE cluster
```

Comment:
Nitpick - please start sentences in the comments with the capital letters for consistency's sake.